test/k8s/updates.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/cilium/cilium/test/helpers"
 )
 
+const (
+	clusterName = "test-upgrade-downgrade"
+	clusterID   = "1"
```

Comment:
Nit. Maybe a larger value (for instance `170`, which translates into `10101010` in binary) would provide an even better coverage.